force-app/main/default/lwc/exportFile/exportFile.js

```diff
@@ -0,0 +1,84 @@
+import { LightningElement, track, wire, api } from 'lwc';
+import { fireEvent, registerListener, unregisterAllListeners } from 'c/pubsub';
+import { CurrentPageReference } from 'lightning/navigation';
+
+
+export default class ExportFile extends LightningElement {
+
+    @wire(CurrentPageReference)
+    pageRef;
+
+    file_options;
+    value;
+
+    payload;
+
+    hasSpinner = false;
+    
+    pdf_options = [
+        { label: 'JPG', value: 'jpg' },
+        { label: 'PNG', value: 'png' }
```

Comment:
@Swoshua  can you add to the `label` here to indicate it will convert to multiple files for multipage docs? ie `PNG (one file per document page)`